code/datums/action.dm

```diff
@@ -19,6 +19,7 @@
 	var/mob/owner
 
 /datum/action/New(Target)
+	START_PROCESSING(SSobj, src)
```

Comment:
For umbrage abilities, I needed it. `has_psi` is required but doesn't check often, so abilities you can't use aren't blacked out and vice-versa. I need something similar to that, but the best I can do is move it to umbrage abilities exclusively.